extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1261,7 +1260,7 @@ export class AmpStoryPage extends AMP.BaseElement {
     }
     this.signals()
       .whenSignal(CommonSignals.LOAD_END)
-      .then(() => this.maybeApplyFirstAnimationFrame())
+      .then(() => this.animationManager_?.applyLastFrame())
```

Comment:
Shouldn't this be `applyFirstAnimationFrameOrFinish`?

Reply:
This is used by `userAgent: bot` which needs to have the last frame of the animation on the page. It's independent of the `prefers-reduced-motion` as it's used to index/crawl the story, so we want all the pages as their final state.